### ISSUE TITLE
[Legitimate Site Blocked] mint.adrunner.io

### ISSUE THREAD

**Issue by Nevkiller13** [MetaMask/eth-phishing-detect#6982]
Please unblock this legitimate site:  https://mint.adrunner.io/

Thank you

**Comment by devenblake** [comment]
Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/6944.
